.ci.yaml

```diff
@@ -4300,7 +4301,7 @@ targets:
           {"dependency": "goldctl", "version": "git_revision:3a77d0b12c697a840ca0c7705208e8622dc94603"}
         ]
       shard: web_tool_tests
-      subshard: web
+      subshard: "1_2"
```

Comment:
uh oh, is this going to work? I'm guessing the test runner is expecting this value to be `"web"`?

Reply:
Looking at this logs https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8801824234378758625/+/u/run_test.dart_for_web_tool_tests_shard_and_subshard_web/test_stdout from a previous run I see `Invalid subshard name "web". Expected format "[int]_[int]" ex. "1_3"` So I think is the other way around; for linux and mac it expects I think 1_1 but since it fails it is running it in a single shard anyways. I can make the change to linux and mac to be `1_1` and avoid that error

Reply:
lol, good catch. i think it's expecting an empty subshard if there's only one. i wonder why we don't fail the build on an invalid subshard value. Anyway, looks good.

Reply:
I agree I think we should fail if invalid subshard name, I spent some time debugging  why my shard was running integration tests and turns out that if you make this mistake  `_selectIndexOfTotalSubshard` returns an empty array which then on `_runDartTest` if you pass an empty array you end up with a command like this `run test --flags` and then it runs every single test on your `workingDirectory`. I will make an issue to fix this